CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v1.x.x (TBD)
+* Add support of distributed training to S3IterableDataset
+
+### Breaking changes
```

Comment:
It is a breaking change for our previous approach, which was not entirely correct. Because of that, I recommend to make new approach as default going forward.